src/Caching/Cache.php

```diff
@@ -98,6 +98,45 @@ public function load($key, $fallback = NULL)
 
 
 	/**
+	 * Reads multiple items from the cache
+	 * @param array
+	 * @param callable
+	 * @return array
+	 */
+	public function bulkLoad(array $keys, $fallback = NULL)
+	{
+		foreach ($keys as $key) {
+			if (!is_scalar($key)) {
+				throw new Nette\InvalidArgumentException('Only scalar keys are allowed in a bulkLoad method.');
+			}
+		}
+		$keys = array_combine(array_map([$this, 'generateKey'], $keys), $keys);
```

Comment:
Replaces whole line with 

~~~php
$storageKeys = array_map([$this, 'generateKey'], $keys);
~~~